### ISSUE TITLE
Render Misskey quote renotes better

### ISSUE THREAD

**Issue by darkphoenix** [glitch-soc/mastodon#1769]
### Pitch

I've searched existing issues because I was kind of surprised it didn't exist already, but how hard would it be to properly render Misskey quote renotes?  Ideally this would include presenting the full quote text in the web UI, making it clickable to open the original thread without jumping into a new tab and going to the original instance, and finding some way to present that to "normal" client apps.

### Motivation

Quote renotes are just plain annoying to use right now - in Tusky at least the link click stays in-app, though it still makes it hard to follow threads with repeated re-notes, and there's always a one-way relationship - you cannot jump from a post to its quote renotes.  I'm not sure how massive the required changes on the server side would be, but as it's at least presented as structured data via federation, the server is in a better position to do it than the client, and it could in the worst case just store it as another reply.  Also, this would enable the server to send a mention to the quoted user if local, avoiding another big usability issue with quote renotes.  As such a feature would almost certainly be rejected by mainline Mastodon due to the projects opinions on quote posts, adding at least visibility (if not the ability to write such posts, I'm still a bit split on whether that's actually desirable myself) seems like the least bad solution here.

**Comment by kescherCode** [comment]
Akkoma has these now as well.
